internal/pkg/validator/validator.go

```diff
@@ -42,7 +42,8 @@ func Validate(value interface{}, rules ...Validation) error {
 	})
 
 	// Do
-	if err := validate.Struct(value); err != nil {
+
+	if err := validate.Var(value, `dive`); err != nil {
```

Comment:
@JakubMatejka , ta chybova hlaska z libky za vela nestoji, ... problem je tu. 
Slice nie je `Struct`. Nikde inde  v kode sme zatial nevalidovali priamo `Slice`,  iba tie vnorene, preto to nebolo spravene univerzalne.

Toto riesenie by malo fungovat pre vsetky typy.

Reply:
Tý jo, díky. 👍  Ha! Takže moje podezření, že `Validate()` nejde volat přímo nad `Inputs` bylo vlastně téměř správný. Začínám se chytat! 😅

Reply:
jj :-) keby si povedal, ze `Struct` nejde volat, tak by to doslo aj mne 😅